### PR TITLE
Tuning pacific/rbd suites - branch name

### PR DESCRIPTION
**Pull request by VasishtaShastry**
rbd tests which consume upstream scripts are failing as branch name of upstream master branch was renamed.
This PR is to change branch name in respective suites.

Apart from this -
- suites/pacific/rbd/tier-2_rbd_regression.yaml - Removed a duplicate test

Signed-off-by: Vasishta <vashastr@redhat.com>

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Comment by rkhambad** [comment]
@VasishtaShastry  can u make changes to tier-0 suite as well for pacific and quicny.

File: suites/nautilus/rbd/tier-1_rbd.yaml
Comment (psathyan) [inline suites/nautilus/rbd/tier-1_rbd.yaml:89]: Could you include the logs for this suite (main and nautilus)?

Comment (VasishtaShastry) [reply suites/nautilus/rbd/tier-1_rbd.yaml]: There seemed to be some pre-existing issue with nautilus upstream script which I did not know, sorry.
`main` is working fine, made changes accordingly 
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1TF8DJ/

**Comment by VasishtaShastry** [comment]
tier-0 using - 
pacific http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-S6DLA0/
main http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-DZEUVO/

Seeing failures in rbd_cli_generic and rbd_cli_migration
Will check and work on it out side of this PR

**Review by ckulal — COMMENTED** [review]
existing changes looks good to me,
But changes also need in quincy suite https://github.com/VasishtaShastry/cephci/blob/ups_brnch_nm/suites/quincy/rbd/basic-operations-unit-testing.yaml

**Comment by VasishtaShastry** [comment]
tier-1
pacific http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-QLJMX1/

**Comment by VasishtaShastry** [comment]
@rkhambad 
made changes to pacific

@rkhambad @ckulal 
Will work on quincy once I address **pacific** upstream script issues as it might affect rc efforts
I think it would be nice if we don't hold this PR for quincy

File: suites/pacific/rbd/tier-0_rbd.yaml
Comment (ckulal) [inline suites/pacific/rbd/tier-0_rbd.yaml:96]: any reason why we changed from node-->nodes

Comment (VasishtaShastry) [reply suites/pacific/rbd/tier-0_rbd.yaml:96]: @ckulal Due to a recent change to tests/cephadm/test_client.py in pull/1666 package installation would be skipped now with old suites.
xxx.xx.92.24/job/rhceph-test-execution-pipeline/574/ - cephfs failure in this run seems to be for the same reason.

Comment (ckulal) [reply suites/pacific/rbd/tier-0_rbd.yaml:96]: Thanks for the clarification

Comment (ckulal) [reply suites/pacific/rbd/tier-0_rbd.yaml:96]: Is it possible to tweak test_client.py (to use both node or nodes), Instead of making changes to all the existing test suites?

Comment (VasishtaShastry) [reply suites/pacific/rbd/tier-0_rbd.yaml:96]: @ckulal sounds good.
as this PR is blocking RC build validation, shall we go ahead with current modifications and track tweaking of test_client.py in issue #1702 ?
(I might not be able to work on modifying test_client.py today)

Comment (ckulal) [reply suites/pacific/rbd/tier-0_rbd.yaml:96]: sure

**Comment by psathyan** [comment]
@VasishtaShastry can split those changes that are needed to unblock for pacific into a separate PR?

**Comment by VasishtaShastry** [comment]
@psathyan I've removed files not required for pacific rc
I've added one more commit to address failures being seen in pacific tier-0 
the fix is being tested can results would be available in 
http://magna002.<mnop>/ceph/cephci-jenkins/cephci-run-NFU98J/ soon

File: suites/pacific/rbd/tier-0_rbd.yaml
Comment (pdhiran) [reply suites/pacific/rbd/tier-0_rbd.yaml:97]: Made changes in the `test_client.py` to accept node/nodes.

Comment (VasishtaShastry) [reply suites/pacific/rbd/tier-0_rbd.yaml:97]: @pdhiran Thanks for the quick fix
Now I need to discard https://github.com/red-hat-storage/cephci/pull/1700/commits/8ebe7e4dfa78f7d346b7060f7369b70bb5e4a795
right ?

Comment (pdhiran) [reply suites/pacific/rbd/tier-0_rbd.yaml:97]: Not a Necessity to discard. It would work with both node/nodes.

**Comment by VasishtaShastry** [comment]
There are two failures in pacific tier-0
cli_generic - jq package was needed from cephci, have addressed it
Got to know that failure we are seeing is expected as downstream version is behind and upstream script if ahead

cli_migration works good as a standalone script
magna002.ceph.redhat.com/cephci-jenkins/cephci-run-LHJFSY

**Comment by VasishtaShastry** [comment]
Update : 
1) Tested again on existing VMs
faced some issues, speculated as cleanup issue
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-AA3BY5/
2) Tried again on fresh run (with new instance creation)
Facing socket timeout issue - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-JXP553/

Will update further

**Comment by VasishtaShastry** [comment]
Working good - 
https://ceph-downstream-jenkins-csb-storage.apps.ocp-c1.prod.psi.redhat.com/view/RHCS%20QE/job/rhceph-test-executor/900/consoleFull

File: suites/nautilus/rbd/tier-1_rbd.yaml
Comment (VasishtaShastry) [reply suites/nautilus/rbd/tier-1_rbd.yaml]: Marking as resolved for now as file no longer exists in the PR.

File: suites/pacific/rbd/tier-0_rbd.yaml
Comment (VasishtaShastry) [reply suites/pacific/rbd/tier-0_rbd.yaml]: Have dropped the respective PR as test_client is ow backward compatible, marking as resolved for now
please feel free to open if its not

File: suites/pacific/rbd/tier-2_rbd_regression.yaml
Comment (VasishtaShastry) [inline suites/pacific/rbd/tier-2_rbd_regression.yaml:105]: Removed this embedded test which was added by mishap as part of PR #1664
The test was used only for kernel fix test.

cc @preethisn

**Review by psathyan — APPROVED** [review]

**Pull request merged**
